tests/tests_basic.yml

```diff
@@ -0,0 +1,7 @@
+---
+# empty playbook to gather facts for import_playbook when clause
+- hosts: all
+
+# workaroud for: https://github.com/ansible/ansible/issues/27973
```

Comment:
Could you put here tl;dr version of the problem?

Reply:
done